astro/src/content/docs/customize/email-and-messages/templates-replacement-variables.mdx

```diff
@@ -11,10 +11,10 @@ import EmailTemplateBaseUrlNote from 'src/content/docs/customize/email-and-messa
 import APIBlock from 'src/components/api/APIBlock.astro';
 import APIField from 'src/components/api/APIField.astro';
 import EnterpriseEditionBlurb from 'src/content/docs/_shared/_enterprise-edition-blurb.astro';
-import BreachedPasswordHTML from 'src/content/docs/_shared/email/_breached-password-html.mdx';
-import BreachedPasswordText from 'src/content/docs/_shared/email/_breached-password-text.mdx';
-import PasswordlessHTML from 'src/content/docs/_shared/email/_passwordless-html.mdx';
-import PasswordlessText from 'src/content/docs/_shared/email/_passwordless-text.mdx';
+import BreachedPasswordHTML from 'src/content/docs/_shared/email/_breached-password-notification-html.mdx';
+import BreachedPasswordText from 'src/content/docs/_shared/email/_breached-password-notification-txt.mdx';
+import PasswordlessHTML from 'src/content/docs/_shared/email/_passwordless-login-html.mdx';
+import PasswordlessText from 'src/content/docs/_shared/email/_passwordless-login-txt.mdx';
```

Comment:
Please update this page to use the remote include components for all the other sample messages.